pennylane/pauli/grouping_utils.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```

Comment:
The logic of the functions in this file have not changed; things have just been renamed to fit in the updated module.